test/dlc_tests/conftest.py

```diff
@@ -975,7 +1027,7 @@ def skip_pt22_test(request):
     if request.node.get_closest_marker("skip_pt22_test"):
         if Version(image_framework_version) in SpecifierSet("==2.2"):
             pytest.skip(
-                f"PT2.2 doesn't support DGL binaries for now, skipping this container with tag {image_framework_version}"
+                f"PT2.2 doesn't support DGL, SMDMP, and SMDDP binaries for now, skipping this container with tag {image_framework_version}"
```

Comment:
nit: This is an unclear skip message based on the fixture name, as the fixture implies that it can be used for any pt22 skip condition, but the message is coupled to DGL, SMDDP and SMMP.